main.py

```diff
@@ -14,13 +14,17 @@
     RANDOM_SEARCH_AGENT,
     AGENT_4o,
     AGENT_4o_MINI,
+    AGENT_o3_MINI,
+    AGENT_o1_MINI,
+    AGENT_37_SONNET,
+    AGENT_CLAUDE_SONNET_35,
 )
 from agentlab.experiments.study import Study
 
 logging.getLogger().setLevel(logging.INFO)
 
 # choose your agent or provide a new agent
-agent_args = [AGENT_4o_MINI]
```

Comment:
was there any intention to swich the agent to o1 mini? Or you inadvertently committed your main.py?